entity-types/ext-snowflake/definition.yml

```diff
@@ -0,0 +1,21 @@
+---
+domain: EXT
+type: SNOWFLAKE
+synthesis:
+  rules:
+  - identifier: hostname
+    name: hostname
+    encodeIdentifierInGUID: true
+    conditions:
+    - attribute: eventType
+      value: snowflakeLongestQueriesSample
+    - attribute: USER_NAME
+      value: "SNOWFLAKETEST"
```

Comment:
Why we are making it fixed to a single username?

Reply:
I have updated, Kindly review it?